File: src/main/java/me/syz/freelook/command/FreelookCommand.java
package me.syz.freelook.command;

import cc.polyfrost.oneconfig.utils.commands.annotations.Command;
import cc.polyfrost.oneconfig.utils.commands.annotations.Main;
import me.syz.freelook.FreelookMod;

@Command(value = FreelookMod.MODID, description = "Access the " + FreelookMod.NAME + " GUI.")
public class FreelookCommand {
    @Main
    private void handle() {
        FreelookMod.config.openGui();
    }
}

File: src/main/kotlin/com/github/chromaticforge/freelook/command/FreelookCommand.kt
package com.github.chromaticforge.freelook.command

import cc.polyfrost.oneconfig.utils.commands.annotations.Command
import cc.polyfrost.oneconfig.utils.commands.annotations.Main
import com.github.chromaticforge.freelook.FreelookMod
import com.github.chromaticforge.freelook.config.FreelookConfig

@Command(
    value = FreelookMod.ID,
    description = "Access the ${FreelookMod.NAME} GUI.",
    aliases = ["perspective", "snaplook"]
)
object FreelookCommand {
    @Main
    private fun main() {
        FreelookConfig.openGui()
    }
}